### PR TITLE
Prefer Phoenix v1.5 defaults when possible

### PR DESCRIPTION
**Pull request by eldano**
Steps taken using `phx_new` 1.5.12:

- `$ cd ..`
- `$ mix phx.new elixir_console --live --no-gettext --no-ecto --no-dashboard`
- Manually chose what to keep and discard

### CODE REVIEW

**Review by grzuy — APPROVED** [review]

File: lib/elixir_console/application.ex [hunk 1]
```diff
@@ -6,12 +6,17 @@ defmodule ElixirConsole.Application do
   use Application
 
   def start(_type, _args) do
-    # List all child processes to be supervised
     children = [
+      # Start the Telemetry supervisor
+      ElixirConsoleWeb.Telemetry,
```
Comment (eldano) [inline lib/elixir_console/application.ex:11]: We aren't really doing anything with Telemetry so it is a bit out of place to include it in this PR. What if we remove it and include it later when we decide to add the live dashboard?

File: assets/yarn.lock [hunk 2]
```diff
@@ -4250,7 +4284,12 @@ semver@^6.3.0:
   resolved "https://registry.yarnpkg.com/semver/-/semver-6.3.0.tgz#ee0a64c8af5e8ceea67687b133761e1becbd1d3d"
   integrity sha512-b39TBaTSfV6yBrapU89p5fKekE2m/NwnDocOVruQFS1/veMgdzuPcnOM34M6CwxW8jH/lxEa5rBoDeUwu5HHTw==
 
-serialize-javascript@^1.4.0, serialize-javascript@^2.1.1, serialize-javascript@^2.1.2:
+serialize-javascript@^1.4.0:
+  version "1.9.1"
+  resolved "https://registry.yarnpkg.com/serialize-javascript/-/serialize-javascript-1.9.1.tgz#cfc200aef77b600c47da9bb8149c943e798c2fdb"
+  integrity sha512-0Vb/54WJ6k5v8sSWN09S0ora+Hnr+cX40r9F170nT+mSkaxltoE/7R3OrIdBSUv1OoiobH1QoWQbCnAO+e8J1A==
```
Comment (eldano) [inline assets/yarn.lock:4290]: This appeared here after removing the resolutions block in package.json, maybe we don't want to do this change without a reason

File: assets/yarn.lock [hunk 3]
```diff
@@ -4300,6 +4339,11 @@ signal-exit@^3.0.0:
   resolved "https://registry.yarnpkg.com/signal-exit/-/signal-exit-3.0.2.tgz#b5fdc08f1287ea1178628e415e25132b73646c6d"
   integrity sha1-tf3AjxKH6hF4Yo5BXiUTK3NkbG0=
 
+signal-exit@^3.0.2:
```
Comment (eldano) [inline assets/yarn.lock:4342]: I wonder why this and the previous resolution aren't resolved by the same version

Comment (eldano) [reply assets/yarn.lock]: I forced this to resolve to the same version by deleting these two entries and running yarn install again

**Pull request merged**